aptos-move/mvhashmap/src/unsync_map.rs

```diff
@@ -0,0 +1,145 @@
+// Copyright © Aptos Foundation
+// SPDX-License-Identifier: Apache-2.0
+
+use crate::{types::MVCodeOutput, utils::module_hash};
+use aptos_crypto::hash::HashValue;
+use aptos_types::{
+    executable::{Executable, ExecutableDescriptor, ModulePath},
+    write_set::TransactionWrite,
+};
+use std::{
+    cell::RefCell,
+    collections::{
+        btree_map::Entry::{Occupied, Vacant},
+        BTreeMap, HashMap,
+    },
+    hash::Hash,
+};
+
+struct Entry<V: TransactionWrite, X: Executable> {
+    // data is Option since we may just use the entry to cache base (storage-version) executable.
+    data: Option<V>,
```

Comment:
add "maybe" to be consistent?

Reply:
Also, data is actually code here, right?
If yes, name is confusing.

Reply:
this no longer exists, much simpler.

Reply:
however UnsyncMap mimics MVHashMap, both modules and data. I renamed everything to entry though.